opentelemetry-sdk/src/opentelemetry/sdk/_metrics/measurement_consumer.py

```diff
@@ -41,15 +46,31 @@ def collect(
 
 
 class SynchronousMeasurementConsumer(MeasurementConsumer):
+    def __init__(self, sdk_config: SdkConfiguration) -> None:
+        self._lock = Lock()
+        self._sdk_config = sdk_config
+        # should never be mutated
+        self._reader_to_state: Mapping[MetricReader, MetricReaderStorage] = {
```

Comment:
```suggestion
        self._metric_reader_to_metric_reader_storage: Mapping[MetricReader, MetricReaderStorage] = {
```
Let's pick one term only, `state` or `storage` and use it consistently.

Reply:
Fixed